examples/metrics/src/main/java/io/opentelemetry/example/metrics/LongCounterExample.java

```diff
@@ -0,0 +1,60 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.LongCounter.BoundLongCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Tracer;
+import java.io.File;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ * Example to search all directories for specific file.
+ * Here we will search directory recursively for specific file, using meter to count searched directories.
+ */
+public class LongCounterExample {
+
+  private static final Tracer tracer =
+      OpenTelemetry.getTracer("io.opentelemetry.example.metrics");
+  private static final Meter sampleMeter = OpenTelemetry.getMeterProvider()
+      .get("io.opentelemetry.example.metrics", "0.5");
+  private static final LongCounter methodCallCounter = sampleMeter
+      .longCounterBuilder("directories_search_count")
+      .setDescription("should count directories searched")
+      .setUnit("unit")
+      .build();
+  //we can use BoundCounters to not specify labels each time
+  private static final BoundLongCounter directoryCounter = methodCallCounter
+      .bind("directory", "searched");
```

Comment:
this is a strange label. Why would this be a useful label for this process?

Reply:
i was thinking to count number of directories that was searched

Reply:
Shouldn't it be like Directories searched 5 with this label?

Reply:
but, the name of the instrument already has this same information in it, so why would you add a redundant label? You might add a bound label for the disk volume, or the host name, or something that would not just repeat the information that is in the instrument name.